src/H5TSprivate.h

```diff
@@ -273,20 +299,31 @@ typedef atomic_flag H5TS_spinlock_t;
 /* Library-private Function Prototypes */
 /***************************************/
 
-#ifdef H5_HAVE_THREADSAFE
+#ifdef H5_HAVE_THREADSAFE_API
 /* Library/thread init/term operations */
-H5_DLL void H5TS_term_package(void);
-H5_DLL int  H5TS_top_term_package(void);
+H5_DLL herr_t H5TS_init(void);
```

Comment:
This routine doesn't seem to be defined anywhere

Reply:
Yes, good catch.